airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -66,7 +66,7 @@
         <Logger name="com.leansoft.bigqueue" level="INFO" />
         <Logger name="io.netty" level="INFO" />
         <Logger name="io.grpc" level="INFO" />
-        <Logger name="io.temporal" level="INFO" />
+        <Logger name="io.temporal" level="WARN" />
```

Comment:
Not working.

Reply:
Are you trying to reduce the logging in the temporal service container or temporal-related logging on other containers? This should only change the level for other containers.

Reply:
I see. Make sense. I will revert this.